Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12317,7 +12317,11 @@ - (void)continueContextMenuInteraction:(void(^)(UIContextMenuConfiguration *))co
 
 #if ENABLE(DATA_DETECTION)
     if ([(NSURL *)linkURL iTunesStoreURL]) {
-        [self continueContextMenuInteractionWithDataDetectors:continueWithContextMenuConfiguration];
```

Comment:
Can we still show a standard link preview in this case?

Reply:
According to https://bugs.webkit.org/show_bug.cgi?id=200485 , we actually can't